CHANGELOG.md

```diff
@@ -1,3 +1,685 @@
+# 1.4.0 (2022-08-26)
+
+## Edge Agent
```

Comment:
I believe there were some fixes for EdgeAgent and EdgeHub to address user creation issues.  Wouldn't those appear here?